app/services/curation_concerns/admin_set_service.rb

```diff
@@ -1,7 +1,11 @@
 module CurationConcerns
   class AdminSetService
-    def initialize(user)
+    attr_reader :user, :context
+    delegate :blacklight_config, :repository, to: :context
+
+    def initialize(user, context)
```

Comment:
If it is, great (and then we can delegate `current_ability` to the context); I had assumed passing in the user was a feature.